shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -336,9 +336,21 @@ void reset() {
         private float[] globalTransform;
         private Rect globalRect;
 
+        void log(String indent) {
+          Log.i(TAG, indent + "SemanticsObject id=" + id + " label=" + label + " actions=" +  actions + " flags=" + flags + "\n" +
+                     indent + "  +-- rect.ltrb=(" + left + ", " + top + ", " + right + ", " + bottom + ")\n" +
```

Comment:
Yes, that's true...  Maybe I'll make it better as I work on it more.  :smiley_cat: